rosdep/python.yaml

```diff
@@ -663,6 +663,16 @@ python-gdown:
   ubuntu:
     pip:
       packages: [gdown]
+python-gevent:
+  fedora:
+    apt:
```

Comment:
apt is not a valid installer on fedora. If you switch to the compact format in my other comment this problem will disappear.